apps/price_pusher/src/evm/pyth-abi.ts

```diff
@@ -0,0 +1,656 @@
+const IPyth = [
```

Comment:
this is a bit annoying because we have to update it when the contract API changes but i guess it's fine...

One thought is that we could update the the pyth EVM SDK to export an object like this, then import that here. That would also help Pyth users who use viem.

Reply:
@cprussin The JSON file is autogenerated from the contract itself.

@jayantk I'll make a separate PR for it because I need to do some clean up there as well.